### ISSUE TITLE
(3.0.0rc02) "docs/opsguide/make install" fails

### ISSUE THREAD

**Issue by deltecent** [ElvishArtisan/rivendell#450]
```
cp: cannot stat ‘sect.rddbconfig.backing_up_and_restoring_the_database.html’: No such file or directory
make[1]: *** [install-exec-local] Error 1
make[1]: Leaving directory `/home/patrick/musicmaster/docs/opsguide'
make: *** [install-am] Error 2
```

**Comment by ElvishArtisan** [comment]
Not seen here.

Have you tried doing 'make maintainer-clean' and then rebuilding the whole thing?

**Comment by deltecent** [comment]
I will try that and let you know. As a work-around I had to remove ```sect.rddbconfig.backing_up_and_restoring_the_database.html``` from ```Makefile.am``` since that section doesn't seem to exist in ```rddbconfig.xml```. After removing that, I can ```make``` and ```make install```, but now ```make``` always recompiles the opsguide.

**Comment by deltecent** [comment]
Same problem after 'make maintainer-clean'. Check if you have a 'docs/opsguide/sect.rddbconfig.backing_up_and_restoring_the_database.html' file and if so remove it and try 'make install'.

**Comment by ElvishArtisan** [comment]
Fixed (I think) in [master a7cf12d]. Please test!

**Comment by deltecent** [comment]
It looks like you fixed it. Thank you!
